service-front/module/Application/src/Services/OpgApiService.php

```diff
@@ -72,6 +72,14 @@ public function getDetailsData(string $uuid): array
     {
         try {
             $response = $this->makeApiRequest('/identity/details?uuid=' . $uuid);
+
```

Comment:
Is the intention to do the front-end refactor as part of another ticket? (makes sense). Has it been created?